src/GraphQl/Type/FieldsBuilder.php

```diff
@@ -253,15 +254,15 @@ private function getResourceFieldConfiguration(string $resourceClass, ResourceMe
                     ];
                 }
 
-                foreach ($resourceMetadata->getGraphqlAttribute($queryName ?? 'query', 'filters', [], true) as $filterId) {
+                foreach ($propertyResourceMetadata->getGraphqlAttribute($queryName ?? 'query', 'filters', [], true) as $filterId) {
                     if (null === $this->filterLocator || !$this->filterLocator->has($filterId)) {
                         continue;
                     }
 
-                    foreach ($this->filterLocator->get($filterId)->getDescription($resourceClass) as $key => $value) {
+                    foreach ($this->filterLocator->get($filterId)->getDescription($className) as $key => $value) {
                         $nullable = isset($value['required']) ? !$value['required'] : true;
                         $filterType = \in_array($value['type'], Type::$builtinTypes, true) ? new Type($value['type'], $nullable) : new Type('object', $nullable, $value['type']);
-                        $graphqlFilterType = $this->convertType($filterType, false, $queryName, $mutationName, $resourceClass, $property, $depth);
+                        $graphqlFilterType = $this->convertType($filterType, false, $queryName, $mutationName, $className, $property, $depth);
```

Comment:
@alanpoulain This parameter seems to not be used inside the `convertType` method?

Reply:
Not used but is useful if you create your own TypeConverter. You have an example in the tests.